nagios/MotechHealth/libexec/TestDbBackupAlert.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
```

Comment:
Why not explicitly use /usr/bin/bash? (or wherever bash is located on this system)